app/league/all/page.tsx

```diff
@@ -42,10 +42,6 @@ const Leagues = (): JSX.Element => {
   };
 
   useEffect(() => {
-    if (!user.id || user.id === '') {
```

Comment:
This entire empty user will be fixed with this change? I had to add it to 2 other pages because that's how it works for now.

Reply:
@shashilo I am confused.  need more info.  I fixed this to remove it from page.tsx and moved that into the useEffect of authContextProvider as that is the only place I could get it to work and the provider wraps around all pages.